### ISSUE TITLE
TypeError: Cannot read property 'toString' of null

### ISSUE THREAD

**Issue by raymatos** [jhuckaby/Cronicle#195]
## Summary

On the scheduled page, clicked on run on a disabled job. Crashed the app.  This was after i made a change to the params with an API request.

## Steps to reproduce the problem

1. Create a job with params
2. create API key
3. update job with API

### Operating system and version?

Ubuntu 18.04

### Node.js version?

v10.16.0

### Cronicle software version?

 0.8.29

## Log Excerpts

```
TypeError: Cannot read property 'toString' of null
    at constructor.launchLocalJob (/opt/cronicle/lib/job.js:504:22)
    at /opt/cronicle/lib/job.js:245:11
    at /opt/cronicle/node_modules/async/dist/async.js:3874:9
    at /opt/cronicle/node_modules/async/dist/async.js:473:16
    at replenish (/opt/cronicle/node_modules/async/dist/async.js:993:25)
    at iterateeCallback (/opt/cronicle/node_modules/async/dist/async.js:983:17)
    at /opt/cronicle/node_modules/async/dist/async.js:958:16
    at /opt/cronicle/node_modules/async/dist/async.js:3871:13
    at /opt/cronicle/lib/job.js:102:6
    at /opt/cronicle/node_modules/pixl-server-storage/list.js:630:7
```

```
[1561700167.539][2019-06-28 05:36:07][instadrone.io][13088][Cronicle][debug][6][Launching local job][{"params":{},"timeout":3600,"catch_up":false,"queue_max":1000,"timezone":"America/New_York","plugin":"pjx8k8gep01","category":"general","algo":"multiplex","multiplex":1,"stagger":0,"retries":0,"retry_delay":0,"detached":0,"queue":0,"chain":"","chain_error":"ejx8o6rls0k","notify_success":"","notify_fail":"","web_hook":"","cpu_limit":0,"cpu_sustain":0,"memory_limit":0,"memory_sustain":0,"log_max_size":0,"notes":"","category_title":"General","group_title":"All Servers","plugin_title":"Stealth Rank","now":1561700167,"source":"Manual (admin)","id":"jjxfnyhdf02","time_start":1561700167.539,"hostname":"instadrone.io","event":"ejx8l0nxj01","event_title":"test","nice_target":"All Servers","command":"REMOVED","log_file":"/opt/cronicle/logs/jobs/jjxfnyhdf02.log"}]
[1561700167.54][2019-06-28 05:36:07][instadrone.io][13088][Cronicle][debug][1][Uncaught Exception: TypeError: Cannot read property 'toString' of null][TypeError: Cannot read property 'toString' of null     at constructor.launchLocalJob (/opt/cronicle/lib/job.js:504:22)     at /opt/cronicle/lib/job.js:245:11     at /opt/cronicle/node_modules/async/dist/async.js:3874:9     at /opt/cronicle/node_modules/async/dist/async.js:473:16     at replenish (/opt/cronicle/node_modules/async/dist/async.js:993:25)     at iterateeCallback (/opt/cronicle/node_modules/async/dist/async.js:983:17)     at /opt/cronicle/node_modules/async/dist/async.js:958:16     at /opt/cronicle/node_modules/async/dist/async.js:3871:13     at /opt/cronicle/lib/job.js:102:6     at /opt/cronicle/node_modules/pixl-server-storage/list.js:630:7]
[1561700167.54][2019-06-28 05:36:07][instadrone.io][13088][Cronicle][debug][1][Emergency Shutdown: TypeError: Cannot read property 'toString' of null][]
```

**Comment by raymatos** [comment]
Looks like i was passing "key" : null for one of the params.

Should you catch this either in the API or the param.toString?

**Comment by jhuckaby** [comment]
Thanks, I'll try to protect against this in the API.  Which param did you set to null, if I may ask?

**Comment by raymatos** [comment]
it was a custom param for a plugin i created.  It didnt matter which one. Here are the updated steps to reproduce.

# Create new plugin
# Add custom param name test with `text` as type
# create new schedule using this plugin
# update the schedule using the API and set the name to `null`
# try to run the schedule

**Comment by jhuckaby** [comment]
Fixed in [v0.8.31](https://github.com/jhuckaby/Cronicle/releases/tag/v0.8.31).  Thanks!
